resources/templates/index.html

```diff
@@ -82,13 +91,21 @@ <h2>Disclaimer<h2>
   </div>
 </div>
 {% end %}
+{% end %}
```

Comment:
I don't know how good your IDE is supporting these templates, but I had a hard time to figure out why there are two of these `end`s on the same level of indentation. If I'm correct the first belongs to the last `if` (which is why you were in need to add it in this PR) and the second (old one) is for the `block content`. Right? Could you eventually try to add a level of indentation for the whole inner part of the `block content` (eventually also the other blocks)? Is this actually possible? I think this could improve the readability and avoids confusions as I experienced.

Reply:
I did it for consistency like this. But I agree that the template directives should get an indentation. Considering that I am doing more refactoring, I will start with the better readable indentation in this file now.